### PR TITLE
[ET-1826] Prevent creating tickets on recurring events

### PR DESCRIPTION
**Pull request by codingmusician**
### 🎫 Ticket

[ET-1826] 
[ET-1827] 

### 🗒️ Description

The functionality to prevent tickets from being created on recurring themes broke due to some changes that were implemented in the `G23.greenpeas` release. This PR fixes that.

### 🎥 Artifacts 

https://www.loom.com/share/3e9008b87c9b4b72a09cc32b18f8cc9a?sid=25396973-2ed9-4e92-adf8-fc1d8bcc6099

### ✔️ Checklist
- [x] I've included a changelog entry in the readme.txt file. 
- [x] My code is tested. 
- [x] My code has [proper inline documentation](https://the-events-calendar.github.io/products-engineering/docs/code-standards/).

[ET-1826]: https://theeventscalendar.atlassian.net/browse/ET-1826?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[ET-1827]: https://theeventscalendar.atlassian.net/browse/ET-1827?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/Tickets/Provider.php [hunk 1]
```diff
@@ -69,29 +68,12 @@ public function register() {
 		$this->container->register( Integrations\Provider::class);
 
 		// RBE only Providers here.
-		$this->register_ct1_providers();
+		$this->container->register_on_action( 'tec_events_custom_tables_v1_fully_activated', ET_CT1_Provider::class );
```
Comment (redscar) [inline src/Tickets/Provider.php:71]: Need to update the `@since` above.

File: src/Tribe/Editor/Warnings.php [hunk 2]
```diff
@@ -56,7 +56,7 @@ public function show_recurring_event_warning_message( $post_id ) {
 			return;
 		}
 
-		$this->render_notice( $this->get_recurring_event_warning_message(), 'info', '#tribe-recurrence-active', 'checked', [ 'recurring_event_warning' ] );
+		$this->render_notice( $this->get_recurring_event_warning_message(), 'info', null, null, [ 'recurring_event_warning' ] );
```
Comment (redscar) [inline src/Tribe/Editor/Warnings.php:59]: Need to update the `@since` above.

**Comment by redscar** [comment]
I'm seeing some linting issues in the npm lint test.

**Comment by codingmusician** [comment]
> I'm seeing some linting issues in the npm lint test.

@redscar Yep. Those have been there awhile and none are for the modifications I introduced. We should probably address those in a separate PR.

**Review by redscar — APPROVED** [review]

**Review by lucatume — APPROVED** [review]

**Pull request merged**